### DISCUSSION TITLE
repos shuffle

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [renovatebot/renovate#18400] [Request Help]
I am having more and more repositories adopting renovate and I am running into an issue, where the repos that are last in the list get less and less attention due to overall limits.

Is there an option that will shuffle or rotate the repos to create more fairness?

**Comment by rarkins** [comment]
Are you using `autodiscover`?

**Reply by ns-ggeorgiev** [reply]
no, I have a list, that grows too long

**Reply by rarkins** [reply]
If you define the list in `config.js`, could you add a simple shuffle function there? e.g. https://javascript.info/array-methods#shuffle-an-array

**Reply by ns-ggeorgiev** [reply]
yes, this will do. I see the .js extension but I just now realized thanks to you that this is just a javascript code.
